src/engine/resources.ts

```diff
@@ -78,6 +80,10 @@ export interface BanishSource extends CombatResource {
   do: Item | Skill;
 }
 
+const commaItem = $items`aquaviolet jub-jub bird, charpuce jub-jub bird, crimsilion jub-jub bird, stomp box`.find((f) =>
+  have(f)
```

Comment:
This will be false because it is only run at the start of the script, and not recomputed after something is pulled. You can just compute it directly in prepare.

Reply:
This was fun to solve. but I solved it.